docs/how_to/hypervisors/openstack.md

```diff
@@ -57,6 +60,7 @@ in order for the Openstack hypervisor to provision hosts properly.
       openstack_tenant: testing
       openstack_network : testing
       openstack_keyname : nopass
+      security_group    : ['sg0', 'default']
```

Comment:
we would prefer it if we not clutter up the basic openstack hosts file with optional settings, because they're hard enough to grok as is.  Please include this as it's own heading under "Openstack-Specific Hosts File Settings." Rather than include a new copy of the hosts file, as done in the user-data section, I think it's sufficient to say that you include the setting in your `CONFIG` section, and it should look like so:

    security_group    : ['sg0', 'default']